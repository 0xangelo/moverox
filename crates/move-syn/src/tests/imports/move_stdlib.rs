use super::*;

test_files!("move-stdlib" {
    address,
    ascii,
    bcs,
    bit_vector,
    bool,
    debug,
    fixed_point32,
    hash,
    macros,
    option,
    string,
    type_name,
    u128,
    u64,
    u8,
    unit_test,
    uq32_32,
    uq64_64,
    vector,
});
