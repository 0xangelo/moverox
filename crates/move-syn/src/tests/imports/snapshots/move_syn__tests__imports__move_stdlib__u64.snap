---
source: crates/move-syn/src/tests/imports/move_stdlib.rs
expression: flat_imports(ast)
---
String -> std::string::String
